package.json

```diff
@@ -18,6 +18,7 @@
     "dependencies": {
         "body-parser": "^1.19.0",
         "express": "^4.17.1",
-        "mongodb": "^3.2.7"
+        "mongodb": "^3.2.7",
```

Comment:
As I understand, mongodb is not needed now

Reply:
done